### PR TITLE
Gamedev Meetup 33 blog post

### PR DESCRIPTION
**Pull request by AngelOnFira**
Still somethings to add, but just putting this initial part up :)

### CODE REVIEW

**Comment by AngelOnFira** [comment]
@ElhamAryanpur @Syn-Nine @zicklag can I get you to check out your sections and add anything you like?

**Comment by ElhamAryanpur** [comment]
Hey man, thank you! All looks good on my end 🚀🚀

**Comment by zicklag** [comment]
Here are the links for bones:

- https://github.com/fishfolk/bones
- https://github.com/fishfolk/jumpy

And a couple recent blog posts that cover the background for a lot of what was shown, if you think that they'd be good to add:

- https://fishfolk.org/blog/jumpy-migration-to-bones-framework/
- https://fishfolk.org/blog/introducing-lua-scripting-in-jumpy/

**Comment by Syn-Nine** [comment]
Looks good to me, thanks!

File: content/blog/rust-gamedev-meetup-33/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+[Galaxion Trade Wars GitHub repository]: https://github.com/Syn-Nine/galaxion-trade-empire/
+[Syn-Nine Twitter]: https://twitter.com/Syn9Dev
+[Syn-Nine Website]: https://syn9.thehideoutgames.com/
+
+### [Bones Engine] | [@zicklag]
+
+> A 'meta-engine' framework made to facilitate the development of moddable,
+> multiplayer 2D games.
+
+[@zicklag]: https://github.com/zicklag
```
Comment (janhohenheim) [inline content/blog/rust-gamedev-meetup-33/index.md:65]:
```suggestion
**Links:**
- [@zicklag]: https://github.com/zicklag
- [Bone Engine GitHub repository]: https://github.com/fishfolk/bones
- [Jumpy GitHub repository]: https://github.com/fishfolk/jumpy
- [Jumpy's Migration to the Bones Framework]: https://fishfolk.org/blog/jumpy-migration-to-bones-framework/
- [Introducing Lua Scripting in Jumpy]: https://fishfolk.org/blog/introducing-lua-scripting-in-jumpy/
```

**Review by janhohenheim — CHANGES_REQUESTED** [review]

File: content/blog/rust-gamedev-meetup-33/index.md (hunk 1 above)
Comment (AngelOnFira) [reply content/blog/rust-gamedev-meetup-33/index.md:65]: Nit: Can we put the links below the section, and link to them? For example, like the section above:

image

Comment (janhohenheim) [reply content/blog/rust-gamedev-meetup-33/index.md:72]: done

**Review by janhohenheim — APPROVED** [review]

**Pull request merged**